src/adb/logcat.js

```diff
@@ -0,0 +1,10 @@
+let PassThrough = require('stream').PassThrough;
+let spawn = require('child_process').spawn;
+
+module.exports = function logcat() {
+  let proc = spawn('adb', ['logcat']);
```

Comment:
Do you think it would work for raptor to assume that multiple devices won't be connected at first? It seems like it would add a good bit of complexity